syntax = "proto3";
package text;

option cc_generic_services = true;

message EchoRequest {
      string message = 1;
};
message EchoResponse {
      string message = 1;
};

message AddRequest {
      int64 a = 1;
      int64 b = 2;
};
message AddResponse {
      int64 res = 1;
};

service EchoService {
      rpc Echo(EchoRequest) returns (EchoResponse);
      rpc add(AddRequest) returns (AddResponse);
};




